src/foundation.cs

```diff
@@ -6183,6 +6263,12 @@ partial interface NSUrl : NSSecureCoding, NSCopying
 		[Static]
 		NSUrl _FromStringRelative (string url, NSUrl relative);
 
+		[Watch (10, 0), TV (17, 0), Mac (14, 0), iOS (17, 0), MacCatalyst (17, 0)]
+		[Static]
+		[Export ("URLWithString:encodingInvalidCharacters:")]
+		[return: NullAllowed]
+		NSUrl FromStringEncoded (string url, bool encodingInvalidCharacters);
```

Comment:
```suggestion
		NSUrl FromString (string url, bool encodingInvalidCharacters);
```

Reply:
The is a reason for this, there is a method already with the same parameters as that same name.

Reply:
There's a constructor with the same parameters, but not a method as far as I can tell?